doc/source/whatsnew/v0.25.1.rst

```diff
@@ -85,7 +85,7 @@ Interval
 Indexing
 ^^^^^^^^
 
--
+- Bug in assignment to multiple columns of a :class:`DataFrame` when some of the columns do not exist (:issue:`13658`)
```

Comment:
this will need a subsection to explain the change. move to 1.0 as this is a non-trivial change of a longstanding issue.

Reply:
I have added a subsection but the docs is failing. Any pointers on what should I change?